configs/maskformer/maskformer_r50_mstrain_16x1_75e_coco.py

```diff
@@ -116,8 +123,15 @@
             dice_cost=dict(
                 type='DiceCost', weight=1.0, pred_act=True, eps=1.0)),
         sampler=dict(type='MaskPseudoSampler')),
-    test_cfg=dict(object_mask_thr=0.8, iou_thr=0.8),
-    # pretrained=None,
+    test_cfg=dict(
+        panoptic_on=True,
+        # the dataset doesn't support the evaluation of semantic segmentation.
+        semantic_on=False,
+        instance_on=False,
+        max_dets_per_image=100,
```

Comment:
We can keep using max_per_image and add comments to indicate that it is for instance segmentation.